### DISCUSSION TITLE
raid 0 for different ssd

### DISCUSSION THREAD

**Discussion by bibiboui** [swar/Swar-Chia-Plot-Manager#298] [Q&A]
hello
 is it interesting to put the ssd disks in raid 0 even if they are not of the same brands or of the same capacity?

thank you for your reply

**Comment by kalosbg** [comment]
It should be possible, but this will help only if you plot one "plot" at a time. If not, with the same success you can trigger 2 separate plotting queues on both disks.

There is one more advantage if the disks are for example less than 956 GB (4 x 239GB-space needed for 32k plot). In that case, you will be able to plot 7 plots at a time instead of 6.

**Comment by swar** [comment]
Please use GitHub discussions for questions
